### DISCUSSION TITLE
Is `forge install` equivalent to `npm install`?

### DISCUSSION THREAD

**Discussion by heysourin** [Cyfrin/foundry-full-course-cu#89] [Q&A] [answered]
When I am forking a JavaScript repo to my local machine, I do `npm install` to install all the dependencies. What would be the equivalent command when I fork a foundry repo that will install all the dependencies?

**Comment by cromewar** [comment, marked as answer]
Hello @heysourin 
When you clone a Foundry repo that you want to work with locally, you can use the following command to build the project and download all the necessary dependencies:

`cargo build`

This command compiles your project and downloads the necessary dependencies defined in the Cargo.toml file which is equivalent to package.json in a Node.js project.

If you just want to fetch and compile dependencies without building the project, you can use:



`cargo fetch`

**Reply by heysourin** [reply]
Thank You!!!
